admin_organization.go

```diff
@@ -117,6 +117,7 @@ func (s *adminOrganizations) List(ctx context.Context, options *AdminOrganizatio
 	return orgl, nil
 }
 
+// List specific organizations in the Terraform Enterprise installation that have permission to use an organization's modules.
```

Comment:
I think the [godoc convention](https://go.dev/blog/godoc) is to begin the comment with the element it describes:

"ListModuleConsumers lists specific organizations..."

Reply:
👆 I noticed there were several comments for functions that don't follow the pattern: `// <FUNCTION> does X and Y` -- something to keep an eye out for.

Reply:
I have not ran into that issue before, but I will keep an eye on that.